MAINTAINERS.yml

```diff
@@ -3717,6 +3721,17 @@ Networking:
   tests:
     - net.ieee802154
 
+"Networking: OCPP":
+  status: maintained
+  maintainers:
+    - ssekar15
```

Comment:
I am not sure if this is accepted, usually new users are not promoted to maintainers directly, but lets see what happens.

Reply:
To my understanding this is accepted for new code base (new driver, or subsys or soc), and I'm already exist on MAINTAINERS.yml file

Reply:
> I'm already exist on MAINTAINERS.yml file

Ah, in that case there should be no problems.